Guide.docc/IncrementalAdoption.md

```diff
@@ -227,3 +227,198 @@ NS_SWIFT_ASYNC_NAME
 NS_SWIFT_ASYNC_NOTHROW
 NS_SWIFT_UNAVAILABLE_FROM_ASYNC(msg)
 ```
+
+## Dispatch
+
+### Ordered work processing in actors, when enqueueing from a synchronous contexts
```

Comment:
This is meant to be "you're used to queue.async{} and thought Task{} does the same, but it doesn't, here's how to get the same order guarantees".